src/libs/actions/PaymentMethods.js

```diff
@@ -311,6 +312,42 @@ function dismissSuccessfulTransferBalancePage() {
     Navigation.navigate(ROUTES.SETTINGS_PAYMENTS);
 }
 
+/**
+ * Looks through each payment method to see if there is an existing error
+ * @param {Object} bankList
+ * @param {Object} cardList
+ * @returns {Boolean}
+ */
+function hasPaymentMethodError(bankList, cardList) {
+    const combinedPaymentMethods = lodashMerge(bankList, cardList);
+    return _.some(combinedPaymentMethods, item => !_.isEmpty(item.errors));
+}
+
+/**
+ * Clears the error for the specified payment item
+ * @param {String} paymentList The onyx key for the provided payment method
```

Comment:
This is a weird param name, if it is named xList, is reasonable to assume it will be an array, but here is really a string for the key. IMO we should change this (same below)

Reply:
Good point, I originally was passing the payment list but then realized I only needed to pass the onyx key - updated